openapi.v2.json

```diff
@@ -745,7 +745,7 @@
         },
         "/h323/devices": {
             "get": {
-                "summary": "List H.323/SIP Devices",
+                "summary": "List H.323/SIP Devices.",
```

Comment:
Should we have a period here, where others do not with quotes?

Reply:
I removed some periods to make it more uniform across the board for period usage. It should now be more uniform but we can add periods back in if you think it looks/reads oddly.